modules/core/src/main/java/org/apache/ignite/internal/processors/task/GridTaskProcessor.java

```diff
@@ -1530,4 +1536,9 @@ else if (msg instanceof GridTaskSessionRequest)
             }
         }
     }
+
+    /** Tries to get task name in appended form(after ', '). */
+    @NotNull private static String tryResolveTaskName(GridTaskWorker<?, ?> task) {
+        return task != null && task.getSession() != null ? (", " + task.getSession().getTaskName()) : "";
```

Comment:
This looks dirty.
I suggest to create a method like
formatMessage(String msg, Nullable GridTaskWorker<?, ?> task)